packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -79,28 +79,28 @@ export const getStyles = memoizeFunction((
     },
 
     thumb: {
-      width: '.5em',
-      height: '.5em',
       borderRadius: '.5em',
       position: 'absolute',
       top: '.2em',
       transition: 'all 0.1s ease',
-      backgroundColor: thumbBackground,
+      borderColor: thumbBackground,
```

Comment:
Let's do this instead: set a border with color transparent on the thumb. FF will automatically choose the right color in HC mode. (You'll probably need box-sizing:border-box as well.)
That way the user can still use bg to set the color, but FF+HC will just automatically work on top of that.